### ISSUE TITLE
Rotate the photo

### ISSUE THREAD

**Issue by mmo1383** [LazZiya/ImageResize#10]
Resizing the image causes it to rotate:

```
 var image = Image.FromFile("wwwroot/imagesProduct/" + file.FileName);
                    var scalefile = ImageResize.Scale(image, 280, 350);
                    scalefile.SaveAs("wwwroot/ImageResize/" + file.FileName);
```

**Comment by LazZiya** [comment]
Can you share the test image?

btw, it is better to re-write your code as below, so the code can dispose the image object automatically after the `using` statement:

````cs
using(var image = Image.FromFile("wwwroot/imagesProduct/" + file.FileName))
{
    image.Scale(280, 350)
         .SaveAs("wwwroot/ImageResize/" + file.FileName);
}
````

**Comment by mmo1383** [comment]
this image
![Uploading 20210715_113951.jpg…]()

**Comment by LazZiya** [comment]
Thank you @mmo1383, I checked your image and it seems the code is ignoring the image rotation. I will publish a hotfix soon.

**Comment by LazZiya** [comment]
fix is done in [v4.1.0](https://www.nuget.org/packages/LazZiya.ImageResize/)

**Comment by mmo1383** [comment]
thank you
